### ISSUE TITLE
bug (es_extended): livery saving

### ISSUE THREAD

**Issue by iSentrie** [esx-framework/esx-legacy#286]
There are 2 types of liveries and at least one isn't saving after recent changes to vehicle properties function.

1. Type
`GetVehicleLivery(vehicle)`

2. Type
I used this for modIndex
`SetVehicleMod(vehicle --[[ Vehicle ]], modType --[[ integer ]], modIndex --[[ integer ]], customTires --[[ boolean ]])`

So this livery set by `GetVehicleMod` native, `modIndex` for livery is `VMT_LIVERY_MOD = 48`

In conclusion `SetVehicleProperties` or `GetVehicleProperties` fails to save/apply this liveries to a vehicle.

https://github.com/esx-framework/esx-legacy/blob/5fa75840e65d625160204f373014f09d3b10bc51/%5Besx%5D/es_extended/client/functions.lua#L831

**Comment by FBFezz** [comment]
Is this fixed in your pr?

**Comment by iSentrie** [comment]
idk, still broken for me even with latest changes

**Comment by Benzo00** [comment]
this is now fixed!
